motor_control/throttle/base_throttle.h

```diff
@@ -0,0 +1,21 @@
+#ifndef BASE_THR_H
+#define BASE_THR_H
+
+class Throttle
+{
+public:
+    Throttle(const int &pin);
+
+    virtual void compute_motor_value() = 0; // this needs to be implemented differently for variable/button throttle
+
+    void read_sensor_value();
+    int get_raw_value() const; // should only be used to debug
+    int get_motor_value() const;
+
+private:
+    int m_raw_value;
+    int m_motor_value;
+    int m_sensor_pin;
+};
+
+#endif
```

Comment:
Such a minor thing. Git has this little warning symbol if you don't have a blank line at the end of a file.

Reply:
Lmao will add it.